orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/DetermineHealthProvidersTask.java

```diff
@@ -98,7 +105,11 @@ public TaskResult execute(Stage stage) {
 
     try {
       String applicationName = (String) stage.getContext().get("application");
-      Moniker moniker = (Moniker) stage.getContext().get("moniker");
+      Moniker moniker = null;
+      if (stage.getContext().containsKey("moniker")) {
+        moniker = objectMapper.convertValue(stage.getContext().get("moniker"), Moniker.class);
```

Comment:
`stage.mapTo("/moniker", Moniker.class)`

Reply:
Oh neat, will do that